docs/pages/blog/mui-x-v8.md

```diff
@@ -0,0 +1,418 @@
+
+The Rich Tree View now supports parent/child selection propagation.
+This means you can select a parent and automatically select all of its children, and vice versa—depending on your requirements and configuration.
+
+<figure>
+  <video autoplay muted loop playsinline style="width: 300px" width="704" height="720">
+    <source alt="Automatic selection of children"  src="/static/blog/mui-x-v8/tree-view-automatic-selection.mp4" type="video/mp4">
+  </video>
+  <figcaption>Automatic selection of children</figcaption>
+</figure>
+
+```tsx
+type TreeViewSelectionPropagation = {
+  descendants?: boolean; // default: false
+  parents?: boolean; // default: false
+};
+```
+
+You can learn more in the [Selection docs](/x/react-tree-view/rich-tree-view/selection/).
+
+### New customization hook
+
+We've officially transitioned to a new developer experience for customizing Tree Items for the Rich Tree View.
+With our new customization hook, you can freely compose your Tree Item while still leveraging all the out-of-the-box features of the Rich Tree View.
+This provides unparalleled flexibility to tailor your tree components to your project's unique requirements.
+
+```js
+const CustomTreeItemComponent = React.forwardRef(function CustomTreeItemComponent(
+  { id, itemId, label, disabled, children }: TreeItemProps,
+  ref: React.Ref<HTMLLIElement>,
+) {
+  const treeItemData = useTreeItem({ id, itemId, children, label, disabled, rootRef: ref });
+
+  return (
+    <TreeItemProvider {...treeItemData.getContextProviderProps()}>
+      <TreeItemRoot {...treeItemData.getRootProps()}>
+        <TreeItemContent {...treeItemData.getContentProps()}>
+          <TreeItemLabel {...treeItemData.getLabelProps()} />
+        </TreeItemContent>
+        {children && <TreeItemGroupTransition {...treeItemData.getGroupTransitionProps()} />}
+      </TreeItemRoot>
+    </TreeItemProvider>
+  );
+}
+```
+
+Learn more this and other options on [Tree Item Customization docs](/x/react-tree-view/tree-item-customization/).
+
+## Charts
+
+### Funnel Charts [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+We have expanded our chart portfolio with the addition of Funnel Charts, providing a new way to visualize data conversions and drop-offs.
+
+<figure>
+  <img width="606" height="300" alt="Funnel charts examples" src="/static/blog/mui-x-v8/funnel-charts.png" loading="lazy">
+  <figcaption>Funnel charts examples</figcaption>
+</figure>
+
+You can learn more in the [Funnel](/x/react-charts/funnel/) docs.
+
+### Radar Charts
+
+Similarly, our Radar Charts offer a dynamic way to display multivariate data, enhancing your users data analysis capabilities.
+
+<figure>
+  <img width="606" height="300" alt="Radar charts examples" src="/static/blog/mui-x-v8/radar-charts.png" loading="lazy">
+  <figcaption>Radar charts examples</figcaption>
+</figure>
+
+You can learn more in the [Radar](/x/react-charts/radar/) docs.
+
+### Server-side rendering for Charts
```

Comment:
It looks like we are missing a proper docs page for this. Typically, I would imagine https://master--material-ui-x.netlify.app/x/react-data-grid/layout/ and add a h2 for SSR. (We will also need to do the same for the data grid).